Lusas_Engine/Convert/ToBHoM/Elements/Bar.cs

```diff
@@ -34,7 +34,7 @@ namespace BH.Engine.Lusas
 {
     public static partial class Convert
     {
-        public static Bar ToBHoMBar(this IFLine lusasLine, 
+        public static Bar ToBar(this IFLine lusasLine, 
```

Comment:
I think this would be better called `FromLusas` (if the convert is from Lusas to BHoM)